### PR TITLE
Use material design icon for appsidebar close

### PR DESCRIPTION
**Pull request by raimund-schluessler**
Uses a material design icon for the appsidebar close button:

After:
Screenshot 2021-09-06 at 21-08-38 Tasks - Nextcloud

Before:
Screenshot 2021-09-06 at 21-10-26 Tasks - Nextcloud

First part of #2199.

### CODE REVIEW

File: src/components/AppSidebar/AppSidebar.vue [hunk 1]
```diff
@@ -596,6 +601,11 @@ $top-buttons-spacing: 6px;
 				opacity: $opacity_full;
 				background-color: $action-background-hover;
 			}
+
+			.material-design-icon {
```
Comment (skjnldsv) [inline src/components/AppSidebar/AppSidebar.vue:605]: Maybe use a dedicated class?

**Review by szaimen — APPROVED** [review]
LGTM

File: src/components/AppSidebar/AppSidebar.vue (hunk 1 above)
Comment (raimund-schluessler) [reply src/components/AppSidebar/AppSidebar.vue]: Done.

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**